include/picongpu/fields/incidentField/profiles/BaseParam.def

```diff
@@ -117,9 +117,9 @@ namespace picongpu
                      *
                      * @{
                      */
-                    static constexpr float_64 FOCUS_POSITION_X_SI = 8.0e-5;
-                    static constexpr float_64 FOCUS_POSITION_Y_SI = 5.0e-5;
-                    static constexpr float_64 FOCUS_POSITION_Z_SI = 8.0e-5;
+                    static constexpr float_64 FOCUS_POSITION_X_SI = 5.0e-5;

```

Comment:
Why are these lines changed? Please revert the all changes in this file.

Reply:
Yes, I suspect the testing for this PR was done using the default param structure and so these parameters were adjusted. But this is not a workflow PIConGPU's user should be doing, so these default values should not make any difference, they are merely for illustration.

Reply:
Reverted.